File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
import java.util.*;
class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int answer = 0;
        
        if(arr1.length == arr2.length) {
            int arr1_sum = Arrays.stream(arr1).sum();
            int arr2_sum = Arrays.stream(arr2).sum();

            answer = arr1_sum == arr2_sum ? 0 : arr1_sum < arr2_sum ? -1 : 1; 
        }else {
            answer = arr1.length < arr2.length ? -1 : 1;
        }
        
        
        return answer;
    }
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    sum1 = sum(arr1)
    sum2 = sum(arr2)
    
    if len(arr1) == len(arr2):
        if sum1 == sum2:
            return 0
        elif sum1 > sum2:
            return 1
        elif sum1 < sum2:
            return -1
    else:
        return 1 if len(arr1) > len(arr2) else -1
            
    
    return "error"